numpy/core/src/multiarray/array_method.h

```diff
@@ -266,6 +267,7 @@ extern NPY_NO_EXPORT PyTypeObject PyBoundArrayMethod_Type;
 #define NPY_METH_contiguous_loop 5
 #define NPY_METH_unaligned_strided_loop 6
 #define NPY_METH_unaligned_contiguous_loop 7
+#define NPY_METH_contiguous_indexed_loop 8
```

Comment:
This also needs to be added in `numpy/core/include/numpy/experimental_dtype_api.h`

As an additional meta-comment (no need for you to address in the context of this PR) it would be nice if all of these loop types had comments in that header explaining the distinction between all these loop types.

Reply:
> This also needs to be added in numpy/core/include/numpy/experimental_dtype_api.h

Done.